### PR TITLE
use released...

### PR DESCRIPTION
**Pull request by nickboldt**
use released /openshift-origin-client-tools-v1.1.4-3941102-windows.zip instead of lastSuccessfulBuild

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -121,7 +120,7 @@ gulp.task('prefetch', function() {
             var fname = body.match(/openshift-origin-client-tools-v\w(\.\w)(\.\w){1,3}-\w{1,3}-\w{8}-\w{7}-windows\.zip/)[0];
             console.log('DOWNLOADING -> ' + currentUrl.concat(fname));
             request(currentUrl.concat(fname))
-              .pipe(fs.createWriteStream(path.join(prefetchFolder, currentKey)));
+              .pipe(fs.createWriteStream(path.join(prefetchFolder, key)));
```
Comment (dgolovin) [inline gulpfile.js]: This doesn't work with callbacks.

**Comment by dgolovin** [comment]
picked one commit with cdk-test changes and released to master.

**Comment by ppitonak** [comment]
@nickboldt for the future, could you please include only related commits in PRs? e.g. this PR is about updating OpenShift client tools but there is also commit "remove ref to electronwinstaller as task no longer exists"

**Comment by nickboldt** [comment]
@ppitonak I can do more PRs if you want but figured that a handful of commits in the same PR would be OK. Sometimes it's hard refrain from fixing typos when I see them, relegating them to a second PR

**Pull request closed without merging**